app/assets/stylesheets/pages/_home.scss

```diff
@@ -1 +1,88 @@
 // Specific CSS for your home-page
+
+
+.container {
+  background-size: cover;
+  background-position: center;
+  padding: 50px 0;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.background {
+  background-color: rgb(246, 235, 222);
+
+  h1 {
+    font-family: 'Playfair Display', serif;
+    font-size: 150px;
+    font-weight: bold;
+    color: black;
+    margin-top: 2px;
+    margin-bottom: 20px;
+  }
+
+  h3 {
+    font-family: 'Playfair Display', serif;
+    font-size: 30px;
+    color: black;
+  }
+
+  .image {
+    opacity: 0.8;
+    border-radius: 8px;
+  }
+}
+
+.name {
+  margin-top: 30px;
+  width: 750px;
```

Comment:
Do you really need to give this div a width ? Does it not have the right one automatically ?
Also, a fixed width isn't very responsive. On a phone you'd still have a div.name of width 750px even though the screen would be less than 400px wide